plugins/Marketplace/stylesheets/plugin-details.less

```diff
@@ -455,9 +459,15 @@
         padding: 24px;
         height: 90px;
         border-top: 1px solid #aaa;
+        margin-top: 1px; // to prevent images overflowing the border
```

Comment:
Unrelated to the change, but I notices some of the floating images were overlapping the border, so this fixes it.